CMakeLists.txt

```
@@ -2219,29 +2224,49 @@ if(FAAD)
   endif()
 endif()
 
+# FDK-AAC is loaded dynamically at runtime by EncoderFdkAac using QLibrary,
+# so copy it into the Windows and macOS packages, but do not link to it.
+find_library(FDK_AAC_LIBRARY fdk-aac)
+# On Windows find_library finds the .lib file, but the installer needs the .dll file.
+find_file(FDK_AAC_DLL fdk-aac.dll PATH_SUFFIXES ${CMAKE_INSTALL_BINDIR})
```

```suggestion
find_file(FDK_AAC_DLL libfdk-aac-2.dll PATH_SUFFIXES bin)
```
This should work. 
see: 
https://github.com/mixxxdj/mixxx/pull/3543/commits/90033594106a2b4f383b06e006cacf058b5e6596

---

This already works. I'm not going to break it with a less sustainable implementation.

---

How ${CMAKE_INSTALL_BINDIR} is "."

---

Ah I see CMAKE_INSTALL_BINDIR i got confused with MIXXX_INSTALL_BINDIR  which is "."

---

The issue is here that it now installs a fdk-aac.dll with an unusual name. The usual name is libfdk-aac-2.dll

---

That is incorrect. DLLs do not conventionally have a `lib` prefix nor a version suffix on Windows. There is nothing to fix here. It works.

---

Yes I know, but we are the first following this convention.